fmriprep/workflows/base.py

```diff
@@ -70,6 +70,7 @@ def init_fmriprep_wf(
     use_bbr,
     use_syn,
     work_dir,
+    coreg_init_header=False,
```

Comment:
We don't pass default values. This was an explicit decision made a long time ago. IIRC it was to avoid silent failures to pass values where appropriate.

```suggestion
    coreg_init_header,
```

Reply:
I don't totally follow the policy here: you don't specify default values in [the API / any / some] function definitions? Where should the default values be coded up?

Reply:
The policy is in response to this possible case: Imagine you set `coreg_init_header=False`, and then some time you call it, you forget to add `coreg_init_header=coreg_init_header`, then it gets disconnected from the option the user passes to the CLI, and tracking it down can be a headache.

The policy applies to the `init_*_wf()` functions, with the exception of things like `name`, which are permissible parameters but don't correspond with high-level knobs on functionality.

In this case the default value is in the CLI argument parser. If what needs to be passed through requires processing into some other value(s) then it might get a bit more complicated.